internal/output/table.go

```diff
@@ -27,32 +25,24 @@ func NewTable(opts ...TableOption) (*Table, error) {
 			return nil, err
 		}
 
-		out = table.pager
+		table.cfg.Out = table.pager
 	}
 
-	table.writer = tabwriter.NewWriter(out, 0, tabWidth, 1, '\t', 0)
-
 	if !table.cfg.NoHeaders {
```

Comment:
do we need this condition?

Reply:
other than this, LGTM

Reply:
Yes. `pterm` does not actually track headers separate from the table data itself so we need to ensure the headers are not written ourselves.